docker/Dockerfile.nightly-cpu

```diff
@@ -0,0 +1,37 @@
+FROM ubuntu:18.04
+
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y \
+    build-essential \
+    libatlas-base-dev \
+    libopencv-dev \
+    graphviz \
+    python-dev \
+    python3-dev \
```

Comment:
We should only install python-dev since python3 doesn't work anyway.

Reply:
This change would require us to modify all our container files... Adding python3 and the required mxnet/mms in python3 (pip3 install) seems to add 300MB in CPU somewhat similar numbers in GPU. I could modify that, but I would like to take that as a separate task..